ProcessMaker/wizard-templates
# Implement Default Launchpad Assets for Guided Templates

This PR introduces default launchpad assets for Guided Templates. It involves updates to the file structure, including the addition of a new directory named `launchpad` within the `assets` directory. This directory contains a subdirectory named `slides`, which stores assets associated with the launchpad carousel. Additionally, the `generate_index.php` script file has been refactored to accommodate the new subdirectory file structure for launchpad assets.

## How to Test

1. Ensure your environment is configured with the correct wizard-template branch by adding `GUIDED_TEMPLATE_BRANCH=task/FOUR-14171` to your .env file.
2. Run the commands `php artisan optimize` and `php artisan processmaker:sync-guided-templates`.
3. Navigate to Processes -> Guided Templates.
4. Run a guided template.
5. Upon completion and redirection to the newly created process, verify that the carousel images are loaded correctly.
6. Run another guided template to ensure there are no image errors on subsequent process imports.

## Related Tickets & Packages

[Core PR](https://github.com/ProcessMaker/processmaker/pull/6250)
[FOUR-14171](https://processmaker.atlassian.net/browse/FOUR-14171)

[FOUR-14171]: https://processmaker.atlassian.net/browse/FOUR-14171?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
